### PR TITLE
Initial import from core

### PR DESCRIPTION
**Pull request by paulmelnikow**
To remove a core dependency in guts-compute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+
+# When unset, use explicit default.
+is_local = get_bool('IS_LOCAL', default=True)
+```
+
+Values are coerced as follows:
+
+- When the variable is unset, or set to the empty string, return `default`.
+- When the variable is set to a truthy value, return `True`.
+  These are the truthy values:
```
Comment (dlsmith) [inline README.md:20]: I think you need a newline or two spaces to trigger the list formatting

File: README.md [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+      - 1
+      - true, yes, on
```
Comment (dlsmith) [inline README.md]: "no"

Comment (dlsmith) [inline README.md]: Doh, nevermind.

File: env_flag.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+       Example falsy values:
+          - 0
+          - no
+    - Ignore case and leading/trailing whitespace.
+    """
+    import os
+    environ_string = os.environ.get(env_var, '').strip().lower()
+    if not environ_string:
+        return default
+    return environ_string in ['1', 'true', 'yes', 'on']
```
Comment (dlsmith) [inline env_flag.py:22]: Should we also include capitalized versions? I.e. check `environ_string.lower() in`?

File: pylintrc [hunk 4]
```diff
@@ -0,0 +1,268 @@
+[MASTER]
```
Comment (dlsmith) [inline pylintrc]: Should we stick this in a separate repo?

File: env_flag.py (hunk 3 above)
Comment (dlsmith) [inline env_flag.py:22]: Dammit. I'm out of it today.

File: Rakefile [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+def command_is_in_path?(command)
+    system("which #{ command} > /dev/null 2>&1")
+end
+
+task :test do
+  raise unless system "nose2"
+end
+
+task :lint do
+  raise unless system "./pylint_test.py baiji --min_rating 10.0"
```
Comment (dlsmith) [inline Rakefile]: Update?

File: pylintrc (hunk 4 above)
Comment (paulmelnikow) [inline pylintrc]: I think that's a good idea. We're starting to have a lot of copies floating around, and I think there is at least one change (trailing whitespace) that hasn't made it to this copy.

I am not sure `pip` is the right tool to install it, though.

Should we just make a rake task to `git clone` it? We can definitely make the repo public.

File: README.md (hunk 1 above)
Comment (paulmelnikow) [inline README.md:20]: Fixed, thanks.

File: Rakefile (hunk 5 above)
Comment (paulmelnikow) [inline Rakefile]: Oof, yes.

File: Rakefile [hunk 6]
```diff
@@ -0,0 +1,58 @@
+$mac_os = `uname -s`.strip == 'Darwin'
+
+desc "Install dependencies for distribution"
+task :install_dist do
+    if $mac_os
+        raise unless system "brew update"
+        raise unless system "brew install pandoc"
```
Comment (dlsmith) [inline Rakefile:22]: We still need the Python module though, right?

Comment (paulmelnikow) [inline Rakefile:22]: That _could_ be why this doesn't work… 😄 

I suppose it's worth taking a moment to get this right.

File: pylintrc (hunk 4 above)
Comment (dlsmith) [inline pylintrc]: Presumably with the pylint script as well? That seems fine.

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except (IOError, ImportError):
+    long_description = ''
+    print 'warning: pandoc or pypandoc does not seem to be installed; using empty long_description'
+
+from setuptools import setup
+
+setup(
+    name='example',
```
Comment (dlsmith) [inline setup.py]: Update?

**Comment by dlsmith** [comment]
👍

File: pylintrc (hunk 4 above)
Comment (paulmelnikow) [inline pylintrc]: Added a rake task, PTAL.

File: Rakefile [hunk 8]
```diff
@@ -0,0 +1,60 @@
+desc "Install style config"
+task :install_style_config do
+    FileUtils.rm_rf "bodylabs-python-style" if Dir.exists? "bodylabs-python-style"
+    raise unless system "git clone --single-branch --depth 1 -b 68_initial https://github.com/bodylabs/bodylabs-python-style.git"
+end
```
Comment (paulmelnikow) [inline Rakefile:14]: @dlsmith

File: Rakefile [hunk 9]
```diff
@@ -1,10 +1,17 @@
+desc "Install style config"
+task :install_style_config do
+    FileUtils.rm_rf "bodylabs-python-style" if Dir.exists? "bodylabs-python-style"
+    raise unless system "git clone --single-branch --depth 1 -b 68_initial https://github.com/bodylabs/bodylabs-python-style.git"
```
Comment (dlsmith) [inline Rakefile]: According to the docs `--single-branch` is implied when you specify `--depth`.

File: Rakefile [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+def command_is_in_path?(command)
+    system("which #{ command} > /dev/null 2>&1")
+end
+
+task :test do
+  raise unless system "nose2"
+end
+
+task :lint do
+  raise unless system "bodylabs-python-style/bin/pylint_test env_flag --min_rating 10.0"
```
Comment (dlsmith) [inline Rakefile:43]: Do you want to warn if the style tools haven't been installed? We'll be able to give a better error message.

File: Rakefile (hunk 9 above)
Comment (paulmelnikow) [inline Rakefile]: Cool, I'll change that.

I think when this switches to a tag the command might need to change…

File: Rakefile (hunk 8 above)
Comment (dlsmith) [inline Rakefile:14]: Looks good

File: Rakefile (hunk 10 above)
Comment (paulmelnikow) [inline Rakefile:43]: Makes sense: fa2482a

**Pull request merged**